tbx/core/blocks.py

```diff
@@ -100,6 +102,28 @@ class Meta:
 class PullQuoteBlock(StructBlock):
     quote = CharBlock(form_classname="quote title")
     attribution = CharBlock()
+    company_logo = ImageChooserBlock(
+        required=False, help_text="Add either a company logo or author image"
+    )
+    author_image = ImageChooserBlock(
+        required=False, help_text="Add either a company logo or author image"
+    )
+    link = LinkBlock(required=False)
```

Comment:
**Suggestion**: Because this isn't just a link but a call to action (with text plus a URL), I'd suggest naming it `call_to_action` (or something in those lines)